elasticsearch/src/test/resources/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0",
+  "defaultSchema": "elasticsearch",
+  "schemas": [
+    {
+      "type": "custom",
+      "name": "elasticsearch",
```

Comment:
No copyright header. No comments explaining its purpose. Looks benign but is broken. 

Would be better to just remove it, if the test can be written without it.